website/blog/2023-08-01-announcing-materialized-views.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Introducing Materialized Views and how to use them"
+description: "In dbt v1.6, we introduce support for materialized views. In this blog post, Amy will review how to use them in your workflow"
+slug: announcing-materialized-views
+
+authors: [amy_chen]
+
+tags: [analytics craft, dbt product updates, data ecosystem]
+hide_table_of_contents: false
+
+date: 2023-08-01
+is_featured: true
+---
+
+## Introduction
+
+The year was 2020. I was a kitten-only household, and dbt Labs was still Fishtown Analytics. A enterprise customer I was working with, Jetblue, asked me for help running their dbt models every 2 minutes to meet a 5 minute SLA.
+
+After getting over the initial terror, we talked through the use case and soon realized there was a better option. Together with my team, I created [lambda views](https://discourse.getdbt.com/t/how-to-create-near-real-time-models-with-just-dbt-sql/1457%20?) to meet the need.
+
+Flash forward to 2023. I’m writing this as my giant dog snores next to me (don’t worry the cats have multiplied as well). Jetblue has outgrown lambda views due to performance constraints (a view can only be so performant) and we are at another milestone in dbt’s journey to support streaming. What. a. time.
```

Comment:
I'm sure the answer is yes but just to doublecheck - Jetblue is ok with us sharing this?

Reply:
I'll check with the account team. I remember a linkedin post about it but should validate

Reply:
got approval from the account team